File: command/admin/eval.py
import discord
from db.function.Querry import *
from config import *

class Eval:
    def __init__(self, message, bot):
        self.bot = bot
        self.message = message
        
    async def run(self):
        if self.message.author.id in OWNER:
            import time
            import os
            command = str(self.message.content[6:])
            try:
                output = eval(command)
                await self.message.channel.send(output)
            except Exception as e:
                error = "`ERROR`\n```py\n{0}```".format(e)
                await self.message.channel.send(error)
                

File: command/admin/eval.js
const Discord = require("discord.js");
require("dotenv").config();
const fs = require("fs");
const Enmap = require("enmap");
const util = require("util");
const CleanText = function (text) {
    if (typeof(text) === "string")
      return text.replace(/`/g, "`" + String.fromCharCode(8203)).replace(/@/g, "@" + String.fromCharCode(8203));
    else
        return text;
}

exports.run = async (message, client, args, lang, querryAsync) => {
        if (process.env.OWNER.split(" ").includes(message.author.id))
        args.shift();
        try {
            var code = args.join(" ");
            let evaled = eval(await code);
            if (typeof evaled !== "string");
            evaled = require("util").inspect(await evaled);
            if (CleanText(await evaled).includes(process.env.TOKEN)) return message.channel.send("T'as cru tu pouvais avoir le token ? Mdr");
            if (CleanText(await evaled).includes("}") && CleanText(await evaled).includes("Packet") && CleanText(await evaled).includes("Row")) message.channel.send(CleanText(await evaled), {code:"xl"});
            else message.channel.send(CleanText(await evaled));
        } catch (err) {
            message.channel.send(`\`ERROR\` \`\`\`xl\n${CleanText(err)}\n\`\`\``);
        };
}
